src/js/service/ImportService.js

```diff
@@ -0,0 +1,121 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
+(function () {
+  /**
+   * @param {!PiskelController} piskelController
```

Comment:
As a more general comment, I've done sort of a lousy job with the JSDoc in this PR: I haven't written descriptions for the methods because several of them were moved wholesale and I'm not sure I understand their whole purpose very well.  That said, I find just annotating types helps me a lot when trying to understand new code.